README.md

```diff
@@ -355,3 +355,42 @@ See [CHANGELOG](CHANGELOG.md) for details.
 
 Thanks to Douglas Crockford for the liberal license for his JSON library, and thanks to
 my employer OpenX and my boss Michael Lum for letting me open source the code.
+
+
+### Releasing
+
+This repository gets released to Maven Central.
+
+- This repository requires JDK 11 to release/test. Without that you might see
+  some tests fail when doing the release preventing you from being able to
+  release.
+- Make sure your `~/.m2/settings.xml` has an entry for `ossrh` server id.
+  ```xml
+  <settings>
+      <!-- ... -->
+      <servers>
+          <!-- Sonatype Maven Release Repository -->
+          <server>
+              <id>ossrh</id>
+              <username><!-- username for Maven Central --></username>
+              <password><!-- password for Maven Central --></password>
+          </server>
+      </servers>
+  </settings>
+  ```
+- Make sure you have a GPG keypair. See
+  https://central.sonatype.org/publish/requirements/gpg/ for more instructions.
+- Since the default branch of this repository cannot be pushed to you need to
+  execute Maven release like so:
+  ```sh
+  ./mvnw release:clean release:prepare -DpushChanges=false
+  ./mvnw release:perform -DlocalCheckout=true
+  ```
+- Once the above finishes you'll have 2 commits locally. Create a new pull
```

Comment:
Typicallly I just merge those - without a PR

Reply:
How? I can't seem to push to the `develop` branch directly.

Reply:
Maybe it is because I have org admin

Reply:
I'll keep the pull request process here then for the time being until I can think of some solution.